app/assets/scripts/views/field-report-form.js

```diff
@@ -469,9 +446,15 @@ class FieldReportForm extends React.Component {
     const result = this.validate();
     if (result) {
       if (step === 5) {
-        console.log('Submit data!!!', this.getSubmitPayload());
-        // this.props._createFieldReport(this.getSubmitPayload());
-        // showGlobalLoading();
+        const payload = this.getSubmitPayload();
+        const userId = _get(this.props.user, 'data.id');
+        if (userId) {
+          payload.user = {pk: userId};
+        } else {
+          console.log('Could not read user ID from state');
```

Comment:
Does it still work without user Id or should this be handled as an error?

Reply:
It would work without a user ID, but that should never happen since this is a logged-in route.